zorg/buildbot/builders/ClangBuilder.py

```diff
@@ -494,103 +500,153 @@ def _getClangCMakeBuildFactory(
-            f.addStep(SetProperty(
-                command=builders_util.getVisualStudioEnvironment(vs, vs_target_arch),
-                extract_fn=builders_util.extractVSEnvironment,
-                env={'CC'  : cc, 'CXX' : cxx}))
-            test_suite_env = Property('vs_env')
-        else:
-            test_suite_env = copy.deepcopy(env)
-            test_suite_env['CC'] = cc
-            test_suite_env['CXX'] = cxx
+                testsuite_fc = InterpolateToPosixPath(
+                               f"-DCMAKE_Fortran_COMPILER=%(prop:builddir)s/{compiler_path}/bin/{fc}")
+                cmake_testsuite += [testsuite_fc,
+                                    "-DTEST_SUITE_FORTRAN:STRING=ON"]
+
+            f.addStep(ShellCommand(name='cmake testsuite',
+                                   command=cmake_testsuite,
+                                   haltOnFailure=True,
+                                   description='cmake testsuite',
+                                   workdir=tsbuild,
+                                   env=env))
 
-        # Steps to prepare, build and run LNT
-        f.addStep(ShellCommand(name='clean sandbox',
-                               command=['rm', '-rf', 'sandbox'],
-                               haltOnFailure=True,
-                               description='removing sandbox directory',
-                               workdir='test',
-                               env=env))
-        f.addStep(ShellCommand(name='recreate sandbox',
-                               command=['virtualenv', '--python=python3', 'sandbox'],
-                               haltOnFailure=True,
-                               description='recreating sandbox',
-                               workdir='test',
-                               env=env))
-        f.addStep(ShellCommand(name='setup lit',
-                               command=[python, lnt_setup, 'develop'],
-                               haltOnFailure=True,
-                               description='setting up LNT in sandbox',
-                               workdir='test/sandbox',
-                               env=env))
-        f.addStep(LitTestCommand(
-                               name='test-suite',
-                               command=test_suite_cmd,
-                               haltOnFailure=True,
-                               description=['running the test suite'],
-                               workdir='test/sandbox',
-                               logfiles={'configure.log'   : 'build/configure.log',
-                                         'build-tools.log' : 'build/build-tools.log',
-                                         'test.log'        : 'build/test.log',
-                                         'report.json'     : 'build/report.json'},
-                               env=test_suite_env))
+            f.addStep(WarningCountingShellCommand(name='build testsuite',
+                                                  command=ninja_cmd,
```

Comment:
This will need to have "-k 0" added so we try to build everything possible (equivalent of https://github.com/llvm/llvm-lnt/pull/25).

But it's neater if we keep that for my PR which will be adding it to the LNT path.

Reply:
This is interesting. For now Clang builder does not have the option to pass -k option to ninja. I am still thinking on whether we should allow buildbots to configure what number tests should fail build before it should stop.
Otherwise we can just hardcode a value such as ninja -k 10 or 100 or more when we run LLVM testsuite.

Reply:
I think if you're anyone other than the committer of the change that broke the one test, `-k 0` is what you'd want. If I'm looking at a build as a maintainer I want to know if there are other failures I need to report.

That said, when we did have thousands of NOEXE, the UI only showed a limited number so `-k 10` might end up equivalent but I'd still say use `-k 0` because then we can look at the results in stdio or json to get the full picture.

Reply:
And now I see you added it, cool.